modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/FillMBLayer.java

```diff
@@ -101,8 +94,8 @@ public Expression getFillColor() {
      * Matches the value of fill-color if unspecified. Disabled by fill-pattern.
      */
     public Expression getFillOutlineColor() {
```

Comment:
The names of the Expression-returning methods are query methods and should leave off the "get" to conform to the convention we discussed for MBLayer.java